ctapipe/io/simteleventsource.py

```diff
@@ -50,12 +50,13 @@
 }
 
 
+NANOSECONDS_PER_DAY = (1 * u.day).to_value(u.ns)
```

Comment:
Does this do any magic beyond `24*60*60*10**9?`

Reply:
no